libmolsketch/bond.cpp

```diff
@@ -231,10 +231,10 @@ namespace Molsketch {
     if (m_bondType != DoubleLegacy) return;
     m_bondType = DoubleSymmetric;
     auto beginBondList = m_beginAtom->bonds();
-    auto beginBonds = QSet<Bond*>(beginBondList.begin(), beginBondList.end());
+    auto beginBonds = QSet<Bond*>::fromList(beginBondList);
```

Comment:
Here we have the same deprecation problem as well. Would be nice to have a macro for that whose definition depends on the Qt version. (the original version used just `beginBondList.toSet()` btw.)